packages/plugins/plugin-kanban/src/components/KanbanContainer.stories.tsx

```diff
@@ -95,7 +101,9 @@ const StorybookKanban = () => {
   const handleTypenameChanged = useCallback(
     (typename: string) => {
       if (kanban?.cardView?.target) {
-        schema?.mutable.updateTypename(typename);
+        invariant(schema);
+        invariant(isMutable(schema));
+        (schema as EchoSchema).updateTypename(typename);
```

Comment:
Not a fan of this. Consider instanceof?

Reply:
this is temporary; we need an API (see TODO on isMutable);